sotodlib/io/bookbinder.py

```diff
@@ -6,6 +6,16 @@
 import numpy as np
 import itertools
 import yaml
+from scipy.interpolate import interp1d
+# 
+class HWPFlags:
```

Comment:
This doesn't add an additional field for flags. In the tod_format docs it says that the `hwp_enc` field should be a float between [0, 2pi), and a negative integer to report any other conditions. So these flags are just used to determine what that negative integer should be.

I actually wasn't sure if these are actually good flags to use and wanted to open it up to a larger discussion, since I'm not so familiar with the HWP data and don't really know what goes into constructing the angle (to be honest I don't really know how "locked" and "stable" should be interpreted). Do you have any suggestions for what flags would be useful here?